src/api/views.py

```
@@ -160,52 +195,81 @@ def parse_bool(val, key):
+        description="Allows an operator to update limited fields of a spot (e.g., 'is_ev', 'is_disabled').",
+        request=SpotOperatorUpdateSerializer,
+        responses={
+            200: SpotOperatorUpdateSerializer,
+            400: OpenApiResponse(ErrorSerializer, description="Validation Error (e.g., trying to change number)"),
+            403: OpenApiResponse(ErrorSerializer, description="Forbidden - Not an operator for this lot"),
+        }
+    )
+    @action(detail=True, methods=["patch"], url_path="operator-update",
+            permission_classes=[IsAuthenticated, IsLotOperator])
+    @transaction.atomic
+    def operator_update(self, request, lot_pk=None, pk=None):
```

Ruff (0.14.2)

Unused method argument: `lot_pk`

(ARG002)